output/output_tools.py

```diff
@@ -0,0 +1,2 @@
+# This file will be used for storing tools (functions) that aren't related to the listed issues on main branch.
+# For example, a function for space parsing of the string commands retrieved from the input team.
```

Comment:
You might also want to reword this sentence for clarity. For example, " for space parsing of the string commands" could be changed to "that parses the spaces in a given command". Additionally, since tools and functions are now defined as two different things, you could also provide an example of a tool that could be stored here.